sleap/nn/training.py

```diff
@@ -20,6 +20,7 @@
 import copy
 
 import sleap
+from sleap import Labels
```

Comment:
Patch coverage is down because I changed all `sleap.Labels` with `Labels`. My feature is tested and uses `Labels` so this shouldn't be a problem.